components/service/glean/src/main/java/mozilla/components/service/glean/storages/TimespansStorageEngine.kt

```diff
@@ -121,6 +121,30 @@ internal open class TimespansStorageEngineImplementation(
         }
     }
 
+    /**
+     * Set the elapsed time explicitly.
```

Comment:
What happens if we mistakenly use this API and the usual API on the same metric? Will it break? Should we take care of that or at least document that the APIs are meant to be mutually exclusive?

Reply:
It won't "break".  Using the `set` API after the `sum` API will override the value.  Using the `sum` API after the `set` API will add to the value.  Do we want / need to provide stricter assurances?  (I agree on making the documentation language stronger on this point).